setup.cfg

```diff
@@ -0,0 +1,28 @@
+[aliases]
+test=pytest
+
+[tool:pytest]
+xfail_strict = true
+addopts =
+    --verbose
+testpaths = test/
+
+[isort]
+line_length = 100
+multi_line_output = 3
+include_trailing_comma = true
+  
+[flake8]
+ignore =
+    E203,  # not pep8, black adds whitespace before ':'
+    E231,  # not pep8, https://www.python.org/dev/peps/pep-0008/#pet-peeves
+    W503,  # not pep8, black adds line break before binary operator
+max_line_length = 100
+max-complexity = 10
+exclude = 
+    __pycache__
+    .tox
+    .git
+    bin
+    build
+    venv
```

Comment:
nit: blank line after

Reply:
Resolving comments added changes in new PR.